systemtest/src/test/java/io/strimzi/systemtest/OpenShiftTemplatesIT.java

```diff
@@ -42,11 +45,15 @@
     @ClassRule
     public static KubeClusterResource cluster = new KubeClusterResource();
 
+    @Rule
+    public JUnitGroupRule rule = new JUnitGroupRule();
+
     private ObjectMapper mapper = new ObjectMapper();
     private Oc oc = (Oc) cluster.client();
     private KubernetesClient client = new DefaultKubernetesClient();
 
     @Test
+    @JUnitGroup(value = {"acceptance"})
```

Comment:
With an array typed annotation member taking a single element you can omit the `{}`. Also when the name of the member is `value` you can omit the name. So this can be written as `@JunitGroup("acceptance")`